### PR TITLE
add upload-log-dir arguments

### PR DESCRIPTION
**Pull request by vshand11**
add upload-log-dir arguments in loader.py and file-copier.py

### CODE REVIEW

**Review by n2iw — CHANGES_REQUESTED** [review]
Please use dashes "-" instead of underscores "_" in the arguments

File: config.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def __init__(self, config_file):
                     self.dataset = config.get('dataset')
                     self.no_parents = config.get('no_parents')
                     self.split_transactions = config.get('split_transactions')
-                    self.upload_log_dir = config.get('upload_log_dir')
+                    self.upload_log_dir = config.get('upload-log-dir')
```
Comment (n2iw) [inline config.py:105]: Please use "**upload_log_dir**" in config file to be consistent with other variables there.

**Review by n2iw — CHANGES_REQUESTED** [review]
I also added a commit to use command line argument to override config file. For historical reason, that is not automatically done in loader.py.

File: config.py (hunk 1 above)
Comment (vshand11) [reply config.py]: Done the change and checked in.  Thanks!

**Pull request merged**